app/controllers/categories_controller.rb

```diff
@@ -48,6 +48,10 @@ def destroy
     end
   end
 
+  def category_history
+    @transactions = current_user.transactions.category_history_transactions(params[:id].to_i)
```

Comment:
@sinpai why you use `to_i` here? `id` always integer.

Reply:
It's a string - http://take.ms/nGVyy. In this case my argument is sensitive to a type, so I do this conversion.